python/lambda-ddb-mysql-etl-pipeline/lambda/dbwrite.py

```diff
@@ -0,0 +1,184 @@
+import json
+import os
+import decimal
+import mimetypes
+import boto3
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+ddb = boto3.resource('dynamodb')
+s3 = boto3.client('s3')
+ddb_table = ddb.Table(os.environ['ASTEROIDS_TABLE'])
+_lambda = boto3.client('lambda')
+
+def handler(event, context):
+    """Retrieve s3 object and write to dynamo or mysql table(not-functional in this example, thus comments)
+
+    Args:
+        event (dict, list, str, int, float, NoneType): Provides event specific data to the handler. It
+            can also be of type list, str, int, float, or NoneType.
+        context (obj): Provides methods and properties that contain invocation, function and
+            execution environment information.
+    Raises:
+        Exception: If file format is not of .json or .csv file types.
+
+    Notes:
```

Comment:
I appreciate what you are trying to do here, but having large blocks of commented out code is not a practice we want to include on our repos.

Could you find a way to include this without the commented out code? (fully implement, or scale down?)

Reply:
@NGL321 no docstrings in general in preferred or mainly additional notes?